tests/api_validation_test.go

```diff
@@ -117,7 +117,7 @@ var _ = Describe("Validation tests", func() {
 			err := utils.DeleteDataVolume(f.CdiClient, f.Namespace.Name, dataVolumeName)
 			Expect(err).ToNot(HaveOccurred())
 		})
-		It("should fail creating an already existing DataVolume", func() {
+		It("[test_id:1030]should fail creating an already existing DataVolume", func() {
```

Comment:
Same, so I understand for new case, we can definitely use qe-tools to genereate, but for old case still same?

Reply:
Okay, after checking your another comment, I see, I think (1030-1033) are fully covered, we can mark my cases to be inactive and generate new one

Reply:
for old TCs we can just use the existing ID
so if now you are going to automated test case number 123,
just add the test_id attribute on your test, it will not create a new one when executing the qe-tools script, but instead will update this TC, with the test automation code (may result in rename, test steps will be overridden with whatever we have in 'by', link to rfe, criticality etc....)

Reply:
BTW, yes, we probably have some duplication now,
if you have a manual test in Polarion, which is actually already automated,
please add 'markfordeletion' tag to it,
i will do a clean up soon and delete all tests with this tag